shuup/core/basket/objects.py

```diff
@@ -242,6 +243,20 @@ def payment_method(self, payment_method):
         self._set_value_to_data("payment_method_id", self.payment_method_id)
 
     @property
+    def extra_data(self):
+        if self._extra_data:
+            return self._extra_data
+        if hasattr(self, "_data"):
+            return self._load().get('extra_data')
+
+    @extra_data.setter
+    def extra_data(self, value):
+        self._extra_data = value
+        # save also on data
+        if value and hasattr(self, "_data"):
```

Comment:
once set, you can not clear the data anymore? I mean, calling `basket.extra_data = None` or even `basket.extra_data = {}` will have no effect at all?